packages/retail-ui-extensions/src/extension-api/navigation-api/navigation-api.ts

```diff
@@ -16,6 +16,13 @@ export interface NavigationApiContent {
    * @param params the parameters you want to pass for the new screen.
    */
   push(screenName: string, params?: any): void;
+
+  /** Opens a full screen with the `Retail::SmartGrid::Modal` extension point.
+   * @param screenName you can specify a screen name, which will be returned in the `NavigationAPI` as `currentScreen`.
+   * This value can be used to evaluate which component should be mounted. The `push(screenName: string)` method in the NavigationAPI can be used then to push
+   * additional screens on top of the stack.
+   */
+  presentModal(screenName?: string): void;
```

Comment:
Sorry I misread this. So this new call will open a modalScreen instead of a fullScreen? 
Do we not pass any params to the modal ?

Reply:
I am probably missing context here but curious to know why we are adding `presentModal` in `navigationApi`? Will it have opportunity for partners to bring more modals within the existing modals?

Reply:
> Sorry I misread this. So this new call will open a modalScreen instead of a fullScreen?

Yeah, it will open a new screen. @samiulmonir06 there is a discussion about this in the dev channel for more context. 

> Do we not pass any params to the modal ?

That's actually a good idea, let me add some. I wonder if `Smart Grid Api` should have an option to send params too. @js-goupil